PROPOSALCOPY.MD

```diff
@@ -0,0 +1,36 @@
+## Proposal to deploy & fund the bot for depositing buffered ether and refund ether spent on gas fee
+
+User's ether is deposited to the Beacon chain staking via depositBufferedEther call on "DepositSecurityModule" smart contract.
+This operation requires significant amount of gas and is performed by the depositor bot.
+
+We propose to fund the depositor bot and refund dev team for gas spent on deposits so far (124.72235422715326 eth)
+
+## Problem
+
+Protocol's APR is earned only on deposited ether - having big buffer hurts APR.
+
+Ether is deposited through the smart contract call (depostiBufferedEther on DepositSecurityModule smart contract 0xDb149235B6F40dC08810AA69869783Be101790e7),
+using signatures of the guardians.
+
+This is a gas-intensive operation, so it should be done balancing gas expenses and the time ether sits in protocol buffer.
+
+We must periodically replenish the bot's wallet to continue doing deposits.
+
+## Proposed solution
+
+Fund the bot that would deposit buffered ether.
+
+#### Gas costs
+
+To reduce gas costs we use special gas strategy.
```

Comment:
Need an estimate: how much ETH is required to deposit 1k ETH buffered? 5k? What's the ballpark for monthly expenses?